data.js

```diff
@@ -6,40 +6,96 @@ export const SCENE_HEIGHT = 1080;
 export const SPRITE_SIZE = 128;
 export const ENEMIES_COUNT_MAX = 2**8;
 export const BULLETS_COUNT_MAX = 2**8;
-export const TIME_MAX = 60 * 30;
-export const ENEMY_SPAWN_INTERVAL = 2;
-export const BULLET_SPAWN_INTERVAL = 1;
 
-export const movementConfig = {
-    playerSpeed: 5,
-    enemiesSpeed: 1,
-};
+export const playerSpeed = 6;
 export const enemyTypes = [
     {
         maxHealth: 1,
         size: 128,
+        speed: 1,
     },
     {
         maxHealth: 2,
         size: 128,
+        speed: 2,
+    },
+    {
+        maxHealth: 8,
+        size: 64,
+        speed: 3.2,
+    },
+    {
+        maxHealth: 12,
+        size: 64,
+        speed: 3.2,
+    },
+    {
+        maxHealth: 32,
+        size: 64,
+        speed: 4,
+    },
+
+    {
+        maxHealth: 1,
+        size: 48,
+        speed: 6,
+    },
+    {
+        maxHealth: 128,
+        size: 32,
+        speed: 2,
     },
     {
         maxHealth: 4,
-        size: 128,
+        size: 32,
+        speed: 8,
     },
     {
-        maxHealth: 8,
-        size: 128,
+        maxHealth: 1,
+        size: 4,
+        speed: 0.2,
+    },
+    {
+        maxHealth: 99999,
+        size: 216,
+        speed: 6,
     },
 ];
 export const bulletTypes = [
     {
         name: "handgun",
         velocity: 1200,
         velocityDamp: 1,
-        reloadTime: 1,
+        reloadTime: 0.5,
         despawnTime: 2,
-        damage: 2,
+        damage: 1,
+        size: 16,
+    },
+    {
+        name: "cannon",
+        velocity: 1600,
+        velocityDamp: 0.97,
+        reloadTime: 8,
+        despawnTime: 32,
+        damage: 8,
+        size: 128,
+    },
+    {
+        name: "pink mine",
+        velocity: 400,
+        velocityDamp: 0.5,
+        reloadTime: 2.1,
+        despawnTime: 600,
+        damage: 3,
+        size: 32,
+    },
+    {
+        name: "accel",
+        velocity: 100,
+        velocityDamp: 1.01,
+        reloadTime: 3.2,
+        despawnTime: 32,
+        damage: 12,
         size: 128,
     },
 ];
```

Comment:
**suggestion**: Consider using a more descriptive name for `gametimeScale` to clarify its purpose.